addons/web/static/src/views/graph/graph_arch_parser.js

```diff
@@ -24,6 +24,11 @@ export class GraphArchParser extends XMLParser {
                     if (node.hasAttribute("cumulated")) {
```

Comment:
This PR allows a line chart in cumulative mode to start at some non zero value if some conditions are met.
For this, the attribute "cumulated_start" should be set to true on the arch root node of the graph view. If set to
true, if the first groupby (if any) is a date/datetime field "d", and if the domain contains a condition based on "d" - thus possibly excluding some groups with "d" values inferior to the least "d" value found in the fetched groups - the initial value for the cumulative char will be set to be the aggregated value for those excluded groups.

Task ID: .....

Reply:
I propose something like that for the commit message.  A PR (doc) should document the new attribute "cumulated_start" at some point

Reply:
the cumulative char --> the cumulative chart

Is there really no task?

Reply:
Yeah it was a NSE request to remove a big readgroup override (see https://github.com/odoo/enterprise/pull/41498) but in the end there was no task created (I can create one if needed but :shrug:)